lib/utils/action-hook-delay.js

```diff
@@ -127,7 +127,7 @@ class ActionHookDelayProcessor extends Emitter {
       this._taskInProgress = makeTask(this.logger, t[0]);
       this._taskInProgress.disableTracing = true;
       this._taskInProgress.exec(this.cs, {ep: this.ep}).catch((err) => {
-        this.logger.info(err, 'ActionHookDelayProcessor#_onNoResponseTimer: error playing file');
+        this.logger.info(err.message, 'ActionHookDelayProcessor#_onNoResponseTimer: error playing file');
```

Comment:
ditto the change above.  Whenever we put a variable in the first part of a log statement please surround it with braces, otherwise the properties merge into the other log message properties and are had to distinguish, ie the above would be
```
this.logger.info({msg: err.message}, 'ActionHookDelayProcessor#_onNoResponseTimer: error playing file');
```
but in this case I would prefer simply
```
this.logger.info(`ActionHookDelayProcessor#_onNoResponseTimer: error playing file: ${err.message}`);
```